<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceMetronome</class>
 <widget class="QWidget" name="InterfaceMetronome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1780</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfaceMetronome</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 211, 239);</string>
  </property>
  <widget class="QPushButton" name="pushButtonEnregistrer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxUnite">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEgale">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>21</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTempo">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>60</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlus">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>70</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMoins">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>70</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/moins.png</normaloff>:/images/moins.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStructure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Structure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSub">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subdivision</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutPrincipale"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>1701</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutSubdivision"/>
  </widget>
  <widget class="QPushButton" name="pushButtonSon">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/son.png</normaloff>:/images/son.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>310</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/jouer.png</normaloff>:/images/jouer.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTaper">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>320</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/TapTempo.png</normaloff>:/images/TapTempo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderVolume">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>230</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolume">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1%</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeparation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>-----------------</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTempo">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>60</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BPM</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonNominateur">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>34</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonDenominateur">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>34</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>1701</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutIndication"/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1770</width>
     <height>1010</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(194, 211, 239);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButtonEnregistrer</zorder>
  <zorder>comboBoxUnite</zorder>
  <zorder>labelEgale</zorder>
  <zorder>lineEditTempo</zorder>
  <zorder>pushButtonPlus</zorder>
  <zorder>pushButtonMoins</zorder>
  <zorder>pushButtonStructure</zorder>
  <zorder>pushButtonSub</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>pushButtonSon</zorder>
  <zorder>pushButtonPlay</zorder>
  <zorder>pushButtonTaper</zorder>
  <zorder>verticalSliderVolume</zorder>
  <zorder>labelVolume</zorder>
  <zorder>labelSeparation</zorder>
  <zorder>pushButtonTempo</zorder>
  <zorder>toolButtonNominateur</zorder>
  <zorder>toolButtonDenominateur</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
